services/QuillLMS/client/app/bundles/Connect/components/home.tsx

```diff
@@ -25,6 +27,20 @@ export const Home = ({playLesson, dispatch}) => {
   const [questionToPreview, setQuestionToPreview] = React.useState<any>(null);
   const [switchedBackToPreview, setSwitchedBackToPreview] = React.useState<boolean>(false);
   const [skippedToQuestionFromIntro, setSkippedToQuestionFromIntro] = React.useState<boolean>(false);
+  const [languageOptions, setLanguageOptions] = React.useState<any>(null);
+  React.useEffect(() => {
+    if (lessons?.hasreceiveddata) {
```

Comment:
Very minor but I would do an early return here, so `if (!lessons?.hasreceiveddata) { return }` and then the rest of the effect.